### DISCUSSION TITLE
Way to get oci8 to work

### DISCUSSION THREAD

**Discussion by aurawindsurfing** [crazywhalecc/static-php-cli#465] [General]
Hi @crazywhalecc 

Thanks for your awesome work! I'm 100% sold on the benefits of static php. Right now I'm trying to find a way around missing oci8 extension. Correct me if I'm wrong but it is installable the issue is that there will be no linking possible with Oracle Instant Client as this can be installed anywhere on the machine. 

Will introduction of variable and compiling as shared not solve that issue? Or forcing user to have Oracle Instant Client in a specific location installed?

Thank you for your time!

**Comment by crazywhalecc** [comment]
The biggest problem is linux does not support dynamic linking in statically linked executables, so it cannot be done on Linux platform.

**Comment by henderkes** [comment]
This is possible now. We support linking against glibc dynamically now.
